src/seedu/addressbook/commands/Command.java

```diff
@@ -43,6 +43,13 @@ public CommandResult execute(){
         throw new UnsupportedOperationException("This method should be implement in child classes");
     }
 
+    /**
+     * judge whether the command will mutate the data
```

Comment:
This method `returns`, not `judges`. Need to rephrase the header comment, but good job for writing it.